README.md

```diff
@@ -260,6 +260,40 @@ Add the following dependency definitions.
 </dependency>
 ```
 
+### Using SNAPSHOT version
+
+To build a branch of the SNAPSHOT version under development, you need to add the repository to `terasoluna-gfw/terasoluna-gfw-parent/pom.xml` or maven's `setting.xml`.
+
+Here is an example of adding it to `setting.xml`.
+
+``` xml
+  <profiles>
+    <profile>
+      ...
```

Comment:
- [ ] Since there is nothing to write between ... is unnecessary.
Same for L286.

Reply:
https://github.com/terasolunaorg/terasoluna-gfw/pull/1384/commits/d4ce12c1fd542ab5abf0416433877eb32d3e20ed
You're right, there's no need to omit <profile>. I've removed it.